### PR TITLE
Limit logging

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

File: lib/gooddata/bricks/middleware/logger_middleware.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def call(params)
           logger = NilLogger.new
         else
           logger = params[:GDC_LOGGER_FILE].nil? ? Logger.new(STDOUT) : Logger.new(params[:GDC_LOGGER_FILE])
-          logger.level = params['GDC_LOG_LEVEL'] if params['GDC_LOG_LEVEL']
+          logger.level = params['GDC_LOG_LEVEL'] || 'info'
```
Comment (yenkins) [inline lib/gooddata/bricks/middleware/logger_middleware.rb]: Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)

File: lib/gooddata/core/logging.rb [hunk 2]
```diff
@@ -92,8 +92,16 @@ def stats_off
       @stats = false
     end
 
+    def log_level
+      if ENV['GD_LOG_LEVEL']
```
Comment (yenkins) [inline lib/gooddata/core/logging.rb]: Calls 'env['gd_log_level']' 2 times (DuplicateMethodCall)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/312/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1158/ : SUCCESS in 2m 49s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1428/ : FAILURE in 2m 27s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/313/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1159/ : SUCCESS in 1m 16s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1429/ : FAILURE in 2m 29s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/319/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1163/ : SUCCESS in 1m 14s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1433/ : SUCCESS in 2m 34s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/91/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1164/ : SUCCESS in 1m 16s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1434/ : SUCCESS in 1m 06s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1453/ : SUCCESS in 14m 49s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/25/ : FAILURE in 36m 41s

**Comment by Hahihula** [comment]
The test failure is just wrongly reported, the tests are passing.

**Pull request merged**